keywords.txt

```diff
@@ -31,6 +31,8 @@ scrollDisplayLeft	KEYWORD2
 scrollDisplayRight	KEYWORD2
 createChar	KEYWORD2
 setRowOffsets	KEYWORD2
+readBFAddr  KEYWORD2
+readRAM KEYWORD2
```

Comment:
```suggestion
readBFAddr	KEYWORD2
readRAM	KEYWORD2
```
The Arduino IDE will only recognize the keyword if you use a true tab as a field separator, not spaces.

Reply:
I did use tabs but my Github's editor settings were changed.
I have corrected them.